Mathlib/Data/Polynomial/Laurent.lean

```diff
@@ -220,6 +231,10 @@ theorem _root_.Polynomial.toLaurent_C (r : R) : toLaurent (Polynomial.C r) = C r
 set_option linter.uppercaseLean3 false in
 #align polynomial.to_laurent_C Polynomial.toLaurent_C
 
+@[simp]
+theorem _root_.Polynomial.toLaurent_comp_C : toLaurent (R := R) ∘ Polynomial.C = C := by
+  ext; simp
```

Comment:
I think it's nice to make it obvious where the interesting version was actually proved here:
```suggestion
theorem _root_.Polynomial.toLaurent_comp_C : toLaurent (R := R) ∘ Polynomial.C = C  :=
  funext Polynomial.toLaurent_C
```